### ISSUE TITLE
Homepage banner image and text fails WCAG contrast

### ISSUE THREAD

**Issue by Fermain** [NoroffFEU/agency.noroff.dev#254]
When the background image is disabled for the home banner, the contrast between the background grey and the text white colour is too low and presents an accessibility risk for those with visual impairments.

When creating a text-image banner it is important to think about cases where the image itself cannot be loaded, such as low bandwidth data connections.
